OrbitGl/CaptureSerializer.cpp

```diff
@@ -100,19 +150,76 @@ ErrorMessageOr<void> CaptureSerializer::Load(const std::string& filename) {
     return Load(file);
   } catch (std::exception& e) {
     ERROR("Loading capture from \"%s\": %s", filename, e.what());
-    return ErrorMessage("Error parsing the capture");
+    return ErrorMessage(
+        "Error parsing the capture\nNote: If the capture "
+        "was taken in a previous Orbit version, it could be incompatible. "
+        "Please check release notes for more information.");
   }
 }
 
+void ReadMessage(google::protobuf::Message* message,
+                 google::protobuf::io::CodedInputStream* input) {
+  uint32_t message_size;
+  input->ReadLittleEndian32(&message_size);
+
+  char* buffer = new char[message_size];
+  input->ReadRaw(buffer, message_size);
+  message->ParseFromArray(buffer, message_size);
+  delete[] buffer;
+}
+
 ErrorMessageOr<void> CaptureSerializer::Load(std::istream& stream) {
-  // Header
-  cereal::BinaryInputArchive archive(stream);
+  google::protobuf::io::IstreamInputStream input_stream(&stream);
+  google::protobuf::io::CodedInputStream coded_input(&input_stream);
+
+  ReadMessage(&header, &coded_input);
+  if (header.version() < MIN_CAPTURE_VERSION) {
+    std::string error_message =
+        "Capture was taken in a previous Orbit version and incompatible with "
```

Comment:
also add %s for versions, we might need a mapping between capture version -> Orbit version.

"This capture was taken with Orbit X.YY which is no longer supported, this version of orbit can load captures starting with Orbit K.NN"